pkg/webui/components/status/index.js

```diff
@@ -42,7 +42,7 @@ const Status = function({ className, status, label, pulse, labelValues, children
   return (
     <span className={classnames(className, style.container)}>
       {statusLabel}
-      <span className={classnames(cls)} />
+      <span className={classnames(cls)} title={status} />
```

Comment:
Keep in mind that the title needs to be translated, so you'd need to also create a message for each status and evaluate that first using `@injectIntl`. Let's introduce a title prop as well, in case one wants to control the title specifically. You could then do: `title={translatedTitle || translatedLabel || translatedStatus}`.

Reply:
Makes sense!